pkg/server/backend_manager.go

```diff
@@ -70,24 +71,17 @@ func GenProxyStrategiesFromStr(proxyStrategies string) ([]ProxyStrategy, error)
 	return ps, nil
 }
 
-type Backend interface {
-	Send(p *client.Packet) error
-	Recv() (*client.Packet, error)
-	Context() context.Context
-}
-
-var _ Backend = &backend{}
-var _ Backend = agent.AgentService_ConnectServer(nil)
+var _ *Backend = &Backend{}
```

Comment:
i'm curious, what is the point of having this instance of the variable?

Reply:
It survives from https://github.com/kubernetes-sigs/apiserver-network-proxy/pull/78

@cheftako do you have an answer?

Reply:
i'm guessing we don't need this, but i'd be happy to see it cleaned up in another PR  to minimize the scope on this PR.

Reply:
@cheftako explained that previously (when Backend was an interface) these deleted lines provided a compile time check that the given types implement that interface. So given this change, it is appropriate to remove.

Reply:
thanks for the extra context @jkh52 !